crates/forge/src/running.rs

```diff
@@ -220,16 +220,16 @@ pub(crate) fn run_test_case(
         ..Default::default()
     };
     let mut cheatable_syscall_handler =
-        CheatableSyscallHandler::wrap(syscall_handler, &mut cheatnet_state);
-    let contract_execution_syscall_handler =
+        CheatableSyscallHandler::wrap(&mut syscall_handler, &mut cheatnet_state);
```

Comment:
That's not that simple, 
since in other file `cairo1_execution` we have the use of syscall handler as well with the `finalize_execution` function from blockifier. That function requires the ownership of the handler, thus it can't be moved to any other wrapper

Reply:
I mean I don't think we need to transfer ownership for any reason here so its fine, I am just not sure why it is a problem? You can move out a field from struct with
```
let z = structure.a;
```

Reply:
Though, transferring the ownership would make possible to move some logic later to a functions `build_testing_syscall_handler` `build_contracts_sycall_handler`

Reply:
And you would avoid a b c d :D